lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -32,6 +32,7 @@ six==1.10.0
 Whoosh==2.7.4
 testfixtures==4.10.0
 galaxy_sequence_utils==1.0.2
+h5py==2.7.0
```

Comment:
I think we need a wheel for this on cargo-port. A start might be here: https://github.com/galaxyproject/starforge/pull/145

Reply:
seems there was also already another PR for this too from a while back: https://github.com/galaxyproject/starforge/pull/44 ..what's the best way forward?

Reply:
The wheels are built upstream and are available in [PyPI](https://pypi.python.org/pypi/h5py/2.7.1) so I don't think anything else should be needed here. I'll copy them to wheels.galaxyproject.org so we have them in case they disappear.

Reply:
Might as well update to `2.7.1` while you're in here.